### PR TITLE
fix(avatar): remove util classes, set tokens

### PR DESCRIPTION
**Pull request by braddialpad**
Vue3 pr https://github.com/dialpad/dialtone-vue/pull/1106

### CODE REVIEW

File: components/tooltip/tooltip_variants.vue
Comment (juliodialpad) [inline components/tooltip/tooltip_variants.vue:1]: This should be included on this PR?

**Review by juliodialpad — APPROVED** [review]
If tooltip changes need to be included here, this looks good, thanks!

File: components/tooltip/tooltip_variants.vue
Comment (braddialpad) [reply components/tooltip/tooltip_variants.vue:1]: hmm I am not sure what is going on with these.. it should have been only the single commit

**Comment by braddialpad** [comment]
fixed

**Pull request merged**